freddytoolapi/Freddy.IntegrationTests/Controllers/CustomerController/GetCustomers.cs

```diff
@@ -41,7 +41,51 @@ public async Task GetAllCustomers_ShouldReturnDataFromDatabase()
                 var result = await _client.GetObjectAsync<CustomerView[]>(url);
 
                 Assert.Equal(2, result.Length);
-                // TODO: Compare with TestData 
+
+                await using (var ctx = _services.CreateDbContext())
+                {
+                    foreach (var customer in result)
+                    {
+                        var findCustomer = await ctx.Customers.FindAsync(customer.Id);
+                        Assert.NotNull(customer);
```

Comment:
mozno by sa zislo spravit nejaky helper/private metodu/whatewer, ktora by vsetky asserty spravila medzi CustomerEntity a CustomerView - samozrejme s nazvom ktory indikuje ze to robi :)

Reply:
gemacht